mne/io/base.py

```diff
@@ -1268,6 +1268,26 @@ def crop(self, tmin=0.0, tmax=None, copy=None):
         return raw
 
     @verbose
+    def segment(self, segment_length=1., verbose=None, **kwargs):
```

Comment:
That's where conversation with @jona-sassenhagen and @Eric89GXL led me. Me and Jona use segmentation quite often and thought it might be useful to expose it, Eric was not very convinced but agreed to add it to `_BaseRaw`.